### PR TITLE
fixed the issue of Toast msg is not available in all languages #740

### PR DESCRIPTION
**Pull request by Sumitmaithani**
**What kind of change does this PR introduce?**
bugfix

**Issue Number:**
Fixes [#740](https://github.com/PalisadoesFoundation/talawa-admin/issues/740)

**Did you add tests for your changes?**
Not required

**Snapshots/Videos:**
塔拉瓦郵報 - Google Chrome 3_18_2023 12_25_30 AM
तलावा भूमिकाएं - Google Chrome 3_18_2023 12_24_58 AM

**Summary**
Updated all toast msg with all languages.

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

**Review by palisadoes — CHANGES_REQUESTED** [review]
The aim of having keys that were English sentences versus versus keys was to make sure that it would be immediately readable without someone constantly having to refer to the JSON files.  This would make programming more efficient.

Please revert the changes so that the original text is restored. Use sentence based keys for any of the new messages you require.

**Comment by Sumitmaithani** [comment]
> The aim of having keys that were English sentences versus versus keys was to make sure that it would be immediately readable without someone constantly having to refer to the JSON files. This would make programming more efficient.
> 
> Please revert the changes so that the original text is restored. Use sentence based keys for any of the new messages you require.

Okay like this right
"Talawa-APIServiceIsUnavailable.IsItRunning?CheckYourNetworkConnectivityToo": "Talawa-API service is unavailable. Is it running? Check your network connectivity too."

**Comment by palisadoes** [comment]
Take guidance from what was there before.

**Comment by palisadoes** [comment]
@sumitra19jha We'll need your opinion on this as it relates to translations

**Comment by palisadoes** [comment]
Please fix the merge conflicts and the tests

**Review by palisadoes — DISMISSED** [review]

**Comment by palisadoes** [comment]
Please fix the conflicting files

**Comment by palisadoes** [comment]
Please fix the failing tests

**Comment by palisadoes** [comment]
Please fix the failing tests.

**Comment by Sumitmaithani** [comment]
@palisadoes I've done all the test cases from 30% to 88.88% only 1.42% remains due to AddOnRegister where add plugin functionality is not added that cause that test case cannot be covered right now but when functionality of plugin added we can covered test cases their also..

**Review by palisadoes — DISMISSED** [review]

**Comment by Sumitmaithani** [comment]
@palisadoes 

This PR can now be merged.

**Comment by palisadoes** [comment]
Please read the Our Pull Request Approval Process comment in its entirety

File: public/locales/en.json [hunk 1]
```diff
@@ -14,7 +14,15 @@
...
     "enterPassword": "Enter Password",
-    "doNotOwnAnAccount": "Do not own an account?"
+    "toastError1": "Talawa-API service is unavailable. Is it running? Check your network connectivity too.",
+    "toastError2": "Captcha Error!",
+    "toastError3": "Please, check the captcha.",
+    "toastError4": "Something went wrong, Please try after sometime.",
+    "toastError5": "Password and Confirm password mismatches.",
+    "toastError6": "Fill all the Details Correctly.",
+    "toastError7": "Sorry! you are not Authorised!",
+    "toastError8": "User not found!",
```
Comment (Rumone) [inline public/locales/en.json:24]: Why is it toastError1..8 and not a more appropriate name based on the string it represents.

File: public/locales/en.json [hunk 2]
```diff
@@ -37,6 +45,8 @@
     "visibleInSearch": "Visible In Search",
     "displayImage": "Display Image",
     "enterName": "Enter Name",
+    "toastSuccess": "Congratulation the Organization is created",
+    "toastError": "Talawa-API service is unavailable. Is it running? Check your network connectivity too.",
```
Comment (Rumone) [inline public/locales/en.json:49]: Why not use a more appropriate name that represent the message. I see toastError as too ambiguous and could lead to potential confusion for future devs.

**Comment by Rumone** [comment]
The main and only issue I have with this PR is the use of inappropriate names for  strings in the localization files.

**Comment by palisadoes** [comment]
Please fix the failing tests. Our test code coverage system will fail if either of these two conditions occur:

1) The overall code coverage drops below the target threshold of the repository
2) Any file in the pull request has code coverage levels below the repository threshold

We do this to ensure the overall reliability of the code base is maintained. This is a prerequisite for getting your PR approved. Assigned reviewers regularly review the PR queue and tend to focus on PRs that are passing.

File: public/locales/en.json (hunk 2 above)
Comment (Rumone) [reply public/locales/en.json:49]: For the message "Talawa-API service is unavailable. Is it running? Check your network connectivity too." The property name `talawaApiUnavailable` should suffice.

Comment (Rumone) [reply public/locales/en.json:49]: Congratulation_the_Organization_is_created could be `createOrgSuccess`

**Comment by palisadoes** [comment]
Please fix the failing tests. Our test code coverage system will fail if either of these two conditions occur:

1) The overall code coverage drops below the target threshold of the repository
2) Any file in the pull request has code coverage levels below the repository threshold

We do this to ensure the overall reliability of the code base is maintained. This is a prerequisite for getting your PR approved. Assigned reviewers regularly review the PR queue and tend to focus on PRs that are passing.

**Comment by Sumitmaithani** [comment]
@palisadoes yes I m working on it.

**Comment by Sumitmaithani** [comment]
I m closing this PR. please review this [PR](https://github.com/PalisadoesFoundation/talawa-admin/pull/845) for this issue.

**Pull request closed without merging**